api/models/auxil/status_setter.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+from django.utils import timezone
+
+from api.models import Coordinator, Group
+from api.models.choices.status import CoordinatorStatus, Status
+from api.models.shared_abstract.person import Person
+
+
+class StatusSetter:
+    @staticmethod
+    def set_status(
+        obj: Group | Person, status: Status, status_since: datetime.datetime | None = None
+    ) -> None:
+        """Sets status, sets `status_since` to current time in UTC, saves object.
+
+        Optionally, pass a datetime object as `status_since` to have identical timestamps
+        for multiple log events.
+        """
+        obj.status = status
+        obj.status_since = status_since or timezone.now()
+        obj.save()
+
+    @staticmethod
+    def set_coordinator_statuses(timestamp: datetime.datetime) -> None:
```

Comment:
IMO its a bit hard to figure out the purpose of this function based on its name and location in the files. Is this supposed to "refresh" coordinator statuses?

Reply:
Maybe I should call it `update_coordinator_statuses` and the class `StatusUpdater` because it basically checks the rules and applies new statuses if the criteria for a different status have been met.

Reply:
yeah that sounds good, maybe even mention coordinators being `working` in the function name to make it clear what kind of update it is